src/Sylius/Bundle/CoreBundle/Tests/Controller/ShippingCategoryControllerTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ShippingCategoryControllerTest extends JsonApiTestCase
```

Comment:
I am wondering if we should name tests like this. The ShippingCategoryController class does not exist and people are used to the fact that there is usually Xyz class and XyzTest. Something that sounds less like a class will be less confusing I hope.

``ShippingCategoryApiTest`` vs ``ShippingCategoryControllerTest``

WDYT?